apps/initializer/lib/initializer.ex

```diff
@@ -11,7 +11,8 @@ defmodule Initializer do
 
     quote location: :keep do
       use GenServer
-      use Retry
+      use Annotated.Retry
```

Comment:
worth it to upgrade the old dep and use the native functionality now?

Reply:
At some point, absolutely. With this PR, absolutely not.